app/src/main/java/com/arygm/quickfix/ui/search/SearchOnBoarding.kt

```diff
@@ -0,0 +1,207 @@
+          SearchCategory(
+              icon = Icons.Outlined.Handyman,
+              title = "Handyman Services",
+              description = "Get help with various minor home repairs and tasks.",
+              onClick = {
+                // Search Handyman
+              }),
+          SearchCategory(
+              icon = Icons.Outlined.CleaningServices,
+              title = "Cleaning Services",
+              description = "Book reliable cleaners for home or office maintenance.",
+              onClick = {
+                // Search Cleaners
+              }),
+          SearchCategory(
+              icon = Icons.Outlined.Carpenter,
+              title = "Carpentry",
+              description = "Hire experienced carpenters for woodwork and construction tasks.",
+              onClick = {
+                // Search Carpenters
+              }),
+          SearchCategory(
+              icon = Icons.Outlined.LocalShipping,
+              title = "Moving Services",
+              description =
+                  "Find professional movers to help with local or long-distance relocation tasks.",
+              onClick = {
+                // Search Movers
+              }))
+
+  BoxWithConstraints {
+    val widthRatio = maxWidth / 411
+    val heightRatio = maxHeight / 860
+    val sizeRatio = minOf(widthRatio, heightRatio)
+
+    // Use Scaffold for the layout structure
+    Scaffold(
+        containerColor = colorScheme.background,
+        topBar = {},
+        content = { padding ->
+          Column(
+              modifier =
+                  Modifier.fillMaxWidth()
+                      .padding(padding)
+                      .padding(top = 40.dp)
+                      .padding(horizontal = 10.dp),
+              horizontalAlignment = Alignment.CenterHorizontally) {
+                Row(
+                    modifier = Modifier.fillMaxWidth().padding(bottom = 10.dp),
+                    horizontalArrangement = Arrangement.Center) {
+                      QuickFixTextFieldCustom(
+                          modifier = Modifier.testTag("searchContent"),
+                          showLeadingIcon = { true },
+                          leadingIcon = Icons.Outlined.Search,
+                          showTrailingIcon = { searchQuery.isNotEmpty() },
+                          trailingIcon = {
+                            IconButton(
+                                onClick = { searchQuery = "" },
+                                modifier = Modifier.testTag("clearSearchQueryIcon")) {
+                                  Icon(
+                                      imageVector = Icons.Filled.Clear,
+                                      contentDescription = "Clear search query",
+                                      tint = colorScheme.onBackground,
+                                  )
+                                }
+                          },
+                          placeHolderText = "Find your perfect fix with QuickFix",
```

Comment:
Like I brought up in an earlier meeting, I feel like we might be overselling the app a bit too much... 😅 
iirc this was agreed upon by the team so if everyone still agrees we can go ahead with the change

cc: @ramshty 

```suggestion
                          placeHolderText = "Find your next QuickFix",
```

Reply:
Yes we agreed on this, i'm ok with this change.